### DISCUSSION TITLE
Why doesn't system background mica and system background acrylic in WindowsApp SDK 1.1 support C++?

### DISCUSSION THREAD

**Discussion by JamasChuang94** [microsoft/WinUI-Gallery#1003] [Q&A]
Why doesn't system background mica and system background acrylic in WindowsApp SDK 1.1 support C++? shouldn't they be synchronized with C#?
Are there any relevant examples? Or do I need to implement it myself using a graphics engine?

**Comment by marcelwgn** [comment]
I can't think of a reason why it shouldn't work with C++. [This article](https://learn.microsoft.com/en-us/windows/apps/desktop/modernize/apply-mica-win32) at least uses C++ and seems to be doing the same thing as the C# sample in the WinUI gallery (though I haven't tried the C++ example).
